core/blockchain.go

```diff
@@ -2112,10 +2112,9 @@ func (bc *BlockChain) insertChain(chain types.Blocks, verifySeals bool) (int, er
 		var followupInterrupt uint32
 		// For diff sync, it may fallback to full sync, so we still do prefetch
 		if len(block.Transactions()) >= prefetchTxNumber {
-			throwaway := statedb.Copy()
 			go func(start time.Time, followup *types.Block, throwaway *state.StateDB, interrupt *uint32) {
 				bc.prefetcher.Prefetch(followup, throwaway, bc.vmConfig, &followupInterrupt)
-			}(time.Now(), block, throwaway, &followupInterrupt)
+			}(time.Now(), block, statedb, &followupInterrupt)
```

Comment:
Would not suggest to share StateDB between Prefetcher and main StateDB.
Use prefetch DB instead, main StateDB can access prefetch DB to get the originStorage

Reply:
Prefetch DB could be used as:  L1.5 Cache

Reply:
The stateDB of main process an prefetcher are different, they only share the sharedpool which store the originStroage maps , I  had adjust the sharedPool to be L1.5 Cache by the new commit
![image](https://user-images.githubusercontent.com/19421226/158939216-e99da91c-d9d6-44d4-88f2-92856866e9e7.png)